docs/references/glossary.md

```diff
@@ -21,79 +21,79 @@ In the context of [transactions](#transaction) on the ledger,
 
 #### actor
 
-An **actor** is a primitive in the [Actor
-Model](https://en.wikipedia.org/wiki/Actor_model). It is a process with
+An **actor** is a primitive in the [actor
+model](https://en.wikipedia.org/wiki/Actor-model). It is a process with
 encapsulated state that communicates with other concurrently running
 actors through asynchronous messages that are received sequentially. The
-Actor Model is relevant to the [IC](#Internet_Computer_(IC))
+actor model is relevant to the [IC](#internet-computer-ic-ic)
 because [canisters](#canister) on the IC (a type of smart
-contract) follow the Actor Model for concurrent and asynchronous
+contract) follow the actor model for concurrent and asynchronous
 computation.
 
 ## B
 
 #### balance
 
-The **balance** of an [ account](#account) on the ledger is
+The **balance** of an [account](#account) on the ledger is
 the sum of all deposits minus the sum of all withdrawals. As a
 degenerate case, it is sometimes useful to say that an account which is
 not present in the ledger has a balance of zero.
 
 The balance of a ledger account is denominated in ICP and is represented
 with eight decimals. Thus, the minimum positive balance of an account is
-0.00000001 or 10^-8 [ ICP](#ICP); this amount is sometimes
+0.00000001 or 10^-8 [ICP](#icp); this amount is sometimes
 referred to as one **e8**.
 
 #### batch
 
-A **batch** is a collection of [ messages](#messages) whose
-order is agreed upon by [ consensus](#consensus).
+A **batch** is a collection of [messages](#messages) whose
+order is agreed upon by [consensus](#consensus).
 
 #### beneficiary
 
-The **beneficiary** of an [ account](#account) is the [
+The **beneficiary** of an [account](#account) is the [
 principal](#principal) who owns the [
 balance](#balance) of the account. The beneficiary of an
 account cannot be changed. The beneficiary of an account may or may not
 be allowed to make [transactions](#transaction) on the
-account (see [ fiduciary](#fiduciary)).
+account (see [fiduciary](#fiduciary)).
 
 #### blockchain
 
 A **blockchain** is a growing list of cryptographically linked blocks,
-agreed upon by [ consensus](#consensus). On the [ Internet
-Computer](#Internet_computer) every [
+agreed upon by [consensus](#consensus). On the [ Internet
```

Comment:
I notice a space here between the bracket and the text. A few lines above, a similar space was removed